crates/runtime/src/ch/messages.rs

```diff
@@ -363,6 +362,31 @@ fn response_query(
             write(&mut blk, qtn.as_str(), tid)?;
             Ok(())
         }
+        Ok(BaseCommandKind::InsertFormatSelectValue(
+            header,
```

Comment:
header for this is unnecessary. header for other format insert is necessary because the CH protocol is two-phase: the data is got from client, client want to verify the schema before it sends out the data.
Just leave this info FYI. I will handle the change!

Reply:
Yeah, I know two-phase. I originally just want to send the header to client so that it can display the action result how many data were affected,  but I checked the flow in the CH, it maybe don't process the logic, it always display zero rows inserted in the result.